### PR TITLE
various fixes for pinentry

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] DESKTOP-128 show typing checkbox
- [x] DESKTOP-127 show typing checkbox defaultValue
- [x] DESKTOP-126 title not changing
- [x] DESKTOP-125 window not resizing with long reason text
- [x] DESKTOP-129 cancel button sending correct error

@keybase/react-hackers 
@patrickxb

### CODE REVIEW

**Comment by cjb** [comment]
:thumbsup:

File: react-native/react/native/pinentry.wrapper.desktop.js [hunk 1]
```diff
@@ -16,6 +16,21 @@ class PinentryWrapper extends Component {
     ipc.send('pinentryNeedProps')
   }
 
+  sendResize () {
+    // Kinda gross but can't figure out a good way to get this working
+    setTimeout(() => {
+      ipc.send('pinentryResize', window.pinentry.scrollHeight)
+    }, 100)
```
Comment (MarcoPolo) [inline react-native/react/native/pinentry.wrapper.desktop.js]: why is this 100?

File: react-native/react/pinentry/index.render.desktop.js [hunk 2]
```diff
@@ -14,6 +14,10 @@ export default class PinentryRender extends Component {
     }
     for (const feature in this.props.features) {
       this.state.features[feature] = this.props.features[feature].defaultValue
+
+      if (feature === 'showTyping') {
+        this.state.showTyping = this.props.features[feature].defaultValue
```
Comment (MarcoPolo) [inline react-native/react/pinentry/index.render.desktop.js]: I think these should be setState, right?

**Comment by MarcoPolo** [comment]
:+1: % Comment

File: react-native/react/pinentry/index.render.desktop.js (hunk 2 above)
Comment (chrisnojima) [inline react-native/react/pinentry/index.render.desktop.js]: actually this should be the constructor, i think it moved around, but i changed it back

**Comment by chrisnojima** [comment]
@patrickxb i merged into your branch

**Pull request merged**